CCPAConsentViewController/Classes/UserConsent.swift

```diff
@@ -72,11 +87,15 @@ import Foundation
         rejectedCategories = try values.decode([String].self, forKey: .rejectedCategories)
         let statusString = try values.decode(String.self, forKey: .status)
         switch statusString {
-            case "rejectedNone": status = .RejectedNone
-            case "rejectedSome": status = .RejectedSome
-            case "rejectedAll":  status = .RejectedAll
-            case "consentedAll": status = .ConsentedAll
-            default: throw MessageEventParsingError(message: "Unknown status string: \(statusString)")
+        case "rejectedNone": status = .RejectedNone
+        case "rejectedSome": status = .RejectedSome
+        case "rejectedAll":  status = .RejectedAll
+        case "consentedAll": status = .ConsentedAll
+        case "RejectedNone": status = .RejectedNone
+        case "RejectedSome": status = .RejectedSome
+        case "RejectedAll":  status = .RejectedAll
+        case "ConsentedAll": status = .ConsentedAll
```

Comment:
Why do we need both camel-case and capital-case statuses (e.g. `rejectedNone` and `RejectedNone`)? To ensure can be decoded/encoded?

Reply:
Here is the scenario,
1. First time we received status as "rejectedNone"
2. Using ConsentStatus enum we are converting that to capital case "RejectedNone"
3. We are storing this value into UserDefaults with encoding.
4. Second time when we load the app we have status "RejectedNone" instead of "rejectedNone" while decoding if we not put the capital case "RejectedNone" then it fails to decode the status Because of this I have to check in both the cases.

Reply:
That makes sense. What if instead of:
> 2. Using ConsentStatus enum we are converting that to capital case "RejectedNone"

we convert it to `rejectNone`?

Reply:
Updated capital-case statuses to camel-case in ConsentStatus enum.